src/os/ui/measurebutton.js

```diff
@@ -163,7 +174,7 @@ os.ui.MeasureButtonCtrl.prototype.getMeasureInteraction_ = function() {
 
 
 /**
- * @param {os.ui.action.ActionEvent} evt The event from the menu
+ * @param {os.ui.menu.MenuEvent<Array<number>>} evt The event from the menu
```

Comment:
The generic type here should be `undefined` to match the menu type (describes the type of the menu context).